app/requests_handlers.py

```diff
@@ -17,7 +17,7 @@
 
 class RequestCommon(View):
 
-    valid_request_types = r'{request_type:\bhousehold-code|individual-code|access-code\b}'
+    valid_request_types = r'{request_type:\bindividual-code|access-code\b}'
```

Comment:
With the deletion of all those tests, we have lost a bit of code coverage of the common code in this file. Line 62 to 68 needs a test or two to cover it .